model/src/main/java/org/cbioportal/model/AlterationEnrichment.java

```diff
@@ -12,14 +12,18 @@
     private String hugoGeneSymbol;
     private String cytoband;
     @NotNull
-    private Integer alteredCount;
+    private Integer alteredInSet1Count;
     @NotNull
-    private Integer unalteredCount;
+    private Integer alteredInSet2Count;
     @NotNull
     private String logRatio;
     @NotNull
     private BigDecimal pValue;
-
+    @NotNull
+    private Integer profiledInSet1Count;
```

Comment:
putting alteredIn and profiledIn together sounds alright to me, but could we leave it as set1Counts and set2Counts rather than a list of counts? that would keep with the pattern in this endpoint of dealing with set1 and set2